Samples/PowerShell/ExampleCmdletsUsage.ps1

```diff
@@ -0,0 +1,47 @@
+<#
+ 
+.COPYRIGHT
+Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+See LICENSE in the project root for license information.
+
+#>
+
+$ModulePath = "H:\publicdw\Intune-Data-Warehouse\Samples\PowerShell\IntuneDataWarehouseCmdlets.psm1"
+
+if (Test-Path "$ModulePath") {
+    Import-Module "$ModulePath"
+}
+else {
+    Write-Host
+    Write-Host "Module Path '$ModulePath' doesn't exist..." -ForegroundColor Red
+    Write-Host
+    break
+}
+
+if (!(Test-IntuneDataWarehouseAuthentication)) {
+
+    Write-Host "To use the Intune Data Warehouse, you must first connect providing a few details..."
+    $User = Read-Host "Enter the UPN of the user to authenticate as"
+    $ApplicationId = Read-Host "Enter the application ID of an AzureAD app that has access to the API"
```

Comment:
Using: "... that has access to get data warehouse information from the Microsoft Intune API"

The AzureAD app exposes the role on Microsoft Intune API as: "Get data warehouse information from Microsoft Intune"